### PR TITLE
🩹  Fix: corrects ipv6 loopback ip shown on UI

### PR DESCRIPTION
**Pull request by techytoes**
Fixes: #1500

### CODE REVIEW

**Comment by ReneWerner87** [comment]
you have only adjusted the tests in the pull request, where is the fix of the problem ?

**Comment by techytoes** [comment]
@ReneWerner87 I have pushed the fix for the problem as well. Let me know if any other changes are also required.

**Comment by ReneWerner87** [comment]
ok, had looked too early, there were only the test data in it

I check now

File: app.go [hunk 1]
```diff
@@ -957,6 +957,11 @@ func (app *App) startupMessage(addr string, tls bool, pids string) {
 		host = "0.0.0.0"
 	}
 
+	if app.config.Network == NetworkTCP6 {
```
Comment (ReneWerner87) [inline app.go:960]: i think we should only set this if the host is not explicitly included in the address used for the listener

please move it in the `if host == "" {` body

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]

File: app.go (hunk 1 above)
Comment (techytoes) [reply app.go:960]: okay. will do this. Thanks!

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**